_rules/SC1-1-1-accessible-name-is-image-file-name.md

```diff
@@ -0,0 +1,81 @@
+---
+name: image accessible name is image file name
```

Comment:
Is the correct spelling "filename" or "file name"? WCAG uses the former, should we use that for consistency? (throughout the rule)